Sources/MapboxNavigation/BottomBannerViewController.swift

```diff
@@ -165,6 +165,14 @@ open class BottomBannerViewController: UIViewController, NavigationComponent {
         distanceRemainingLabel.text = "4 mi"
         arrivalTimeLabel.text = "10:09"
     }
+
+    open func reinstallConstraints() {
```

Comment:
I wonder why we made it open in first place. Looks like it's purpose is to be used only internally.